servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcUtils.java

```diff
@@ -349,7 +349,7 @@ private static void ensureGrpcStatusReceived(final HttpHeaders headers) {
                                             final List<T> supportedEncoders,
                                             final Function<T, CharSequence> messageEncodingFunc) {
         final CharSequence encoding = headers.get(GRPC_MESSAGE_ENCODING_KEY);
-        if (encoding == null) {
+        if (encoding == null || contentEqualsIgnoreCase(Identity.identityEncoder().encodingName(), encoding)) {
```

Comment:
nit: consider static import

Reply:
I would love to, but this method already has `identityEncoder` variable